lib/schemaToQueryTree.js

```diff
@@ -61,6 +84,20 @@ function getKeys(field) {
   return queries;
 }
 
+/**
+ * @example
+ *   magiclyExtractFieldTypeName({ type: { name: 'Person' } }) // => 'Person'
+ *   magiclyExtractFieldTypeName({ 
```

Comment:
for second example maybe non-nullable list of string would be a better example than NotMe of MeNeither of TestType?